src/test/ui/mir-dataflow/indirect-mutation-offset.rs

```diff
@@ -1,5 +1,7 @@
 // compile-flags: -Zunleash-the-miri-inside-of-you
 
+// ignore-test Temporarily ignored while this analysis is migrated to the new framework.
```

Comment:
wow look at that, an ignored test came back! How often does that happen around here?! :)